app/controllers/declaration/OfficeOfExitController.scala

```diff
@@ -67,18 +67,23 @@ class OfficeOfExitController @Inject()(
       )
   }
 
-  private def nextPage(declarationType: DeclarationType, answer: String): Mode => Call =
+  private def nextPage(declarationType: DeclarationType, answer: String)(implicit request: JourneyRequest[AnyContent]): Mode => Call =
     if (answer == AllowedUKOfficeOfExitAnswers.no) {
       controllers.declaration.routes.OfficeOfExitOutsideUkController.displayPage
     } else {
       declarationType match {
         case DeclarationType.SUPPLEMENTARY | DeclarationType.STANDARD =>
           controllers.declaration.routes.TotalNumberOfItemsController.displayPage
+        case DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL | DeclarationType.CLEARANCE if (previousDocumentsProvided()) =>
```

Comment:
Same here.  I don't think logic about PreviousDocuments should be leaking out to other controllers

Reply:
Done